src/verify-valid-input-format.js

```diff
@@ -32,6 +32,11 @@ exports.verifyInputFormatForServices = function (services) {
       errorsToReport.push(
         `Invalid import map in request body -- module with name '${moduleName}' does not have a string url`
       );
+    } else if (!moduleName) {
```

Comment:
Maybe we could check for blank strings too, here?

```suggestion
    } else if (moduleName.trim().length === 0) {
```